pkg/cmd/cli/cmd/newapp.go

```diff
@@ -612,7 +612,13 @@ func handleRunError(err error, baseName, commandName, commandPath string) error
 		transformError(err, baseName, commandName, commandPath, groups)
 	}
 	buf := &bytes.Buffer{}
+	fmt.Fprintf(buf, "Prior to API Object processing, here were the results from application argument classification:\n")
```

Comment:
only print this if len(config.ArgumentClassificationErrors)>0
(in conjunction w/ not adding things to that map unless the value is non-nil, as mentioned below)